### ISSUE TITLE
traccc-data module access is ssh on purpose ?

### ISSUE THREAD

**Issue by chavid** [acts-project/traccc#12]
Is there a reason why the traccc-data module is accessed through a ssh cern gitlab address ? An https access would make it more easily accessible by non-cern partners.

**Comment by asalzburger** [comment]
I *think* you can access it also via `https://`as well, right?

As discussed though, it does not have to live on cern gitlab, it was simply as I was sure we have git-lfs support there.

**Comment by chavid** [comment]
Yes, we can. Provided we restart from scratch. Once one has done the first `git submodule update --init`call, it seems the adress from `.gitmodules` is cached somewhere. I am not fluetn with git modules. Th eimportant point is to change the address in `.gitmodules` before calling `git submodule update --init`. I would suggest to change for `https://` the default value provided in `.gitmodules`, but it is not a big matter. I understand the point about `git-lfs`.

**Comment by HadrienG2** [comment]
Fixed by #51
